### PR TITLE
Make code climate happy

### PR DESCRIPTION
**Pull request by indrekj**
AS-4

### CODE REVIEW

File: lib/freddy/consumers/respond_to_consumer.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
+require_relative 'consumer'
+
 class Freddy
   module Consumers
-    class RespondToConsumer
+    class RespondToConsumer < Consumer
```
Comment (urmastalimaa) [inline lib/freddy/consumers/respond_to_consumer.rb]: I don't see a reason for bringing inheritance into the mix.
I think that a simple ReceiveLogger object which takes @logger as an argument is better.

**Comment by stenver** [comment]
No new tests or old changed?

**Comment by indrekj** [comment]
@urmastalimaa added a commit, not exactly as you said, but something similar

**Comment by indrekj** [comment]
@stenver No need, no new features

**Comment by deiwin** [comment]
+1

**Comment by stenver** [comment]
Ahh, yes, now I see it was mostly moving around stuff

**Pull request merged**